lib/fathead/numpy/parse.py

```diff
@@ -0,0 +1,344 @@
+
+    title = get_title(section_div)
+
+    if not title:
+        return ""
+
+    abstract = get_abstract(section_div)
+
+    if not abstract:
+        return ""
+
+    related_topics = get_related_topics(section_div)
+
+    redirects = get_redirects(title)
+
+    url = DOCS_URL + op.basename(doc_file)
+
+    output_entries = OUTPUT_TEMPLATE.format(title=title,
+                                            entry_type="A",
+                                            redirect_title="",
+                                            empty_field="",
+                                            categories="",
+                                            related_topics=related_topics,
+                                            external_links="",
+                                            disambiguation="",
+                                            image="",
+                                            abstract=abstract,
+                                            url=url)
+
+    for redirect in redirects:
+        output_entries = output_entries \
+                + "\n" \
+                + OUTPUT_TEMPLATE.format(title=redirect,
+                                         entry_type="R",
+                                         redirect_title=title,
+                                         empty_field="",
+                                         categories="",
+                                         related_topics="",
+                                         external_links="",
+                                         disambiguation="",
+                                         image="",
+                                         abstract="",
+                                         url="")
+
+    return output_entries + "\n"
+
+
+def get_title(section_div):
+    """Return the fathead entry title.
+
+    Parameters
+    ----------
+    section_div : bs4.BeautifulSoup
+        The BeautifulSoup object corresponding to the div with the "class"
+        attribute equal to "section" in the html doc file.
+
+    Returns
+    -------
+    title : Str
+        The fathead entry title.
+    """
+    title = section_div.h1.text[:-1]
+    return title
+
+
+def get_abstract(section_div):
+    """Return the fathead entry abstract.
+
+    Parameters
+    ----------
+    section_div : bs4.BeautifulSoup
+        The BeautifulSoup object corresponding to the div with the "class"
+        attribute equal to "section" in the html doc file.
+
+    Returns
+    -------
+    abstract : Str
+        The fathead entry abstract.
+    """
+    try:
+        information = section_div.dd.p.text
+    except AttributeError:
+        information = ""
+    if not information:
+        return ""
+
+    rendered_information = INFORMATION_TEMPLATE.format(information=information)
+
+    raw_signature = [each.string for each in section_div.dt.children]
+    signature = ''.join(raw_signature).strip().replace("¶", "")
```

Comment:
We should remove `\t` here instead of escaping it later. I don't think `\t` is necessary for he description text.

We may also want to remove the `\n` -- I'm not sure how much of a difference they make?